images/build/debian-base/buster/Dockerfile.build

```diff
@@ -27,6 +26,16 @@ COPY excludes /etc/dpkg/dpkg.cfg.d/excludes
 # Convenience script for building on this base image.
 COPY clean-install /usr/local/bin/clean-install
```

Comment:
Should we use --chmod here, we were seeing some failures on s390x. See https://github.com/kubernetes/release/pull/1684#discussion_r573982424

Reply:
alternatively: Kubernetes fixed this upstream with `umask 0022` in the build scripts and / or chmod on the host (one less layer / don't need to run it N times for each image).

Reply:
Yes, I decided to add a `RUN chmod +x /usr/local/bin/clean-install` to stay a bit more backwards compatible. WDYT?

Reply:
more recent: (umask)
-  https://github.com/kubernetes/kubernetes/pull/90994
- https://github.com/kubernetes/kubernetes/pull/90779
what I remembered: (chmod on host)
-  https://github.com/kubernetes/kubernetes/pull/67222

Reply:
Set the umask in the `Makefile` now, this should address the issue.